jekyll/_cci2/env-vars.md

```diff
@@ -422,6 +422,8 @@ jobs:
 
 The following example shows separate environment variable settings for the primary container image (listed first) and the secondary or service container image.
 
+**Note**: While hard coded environment variable values will be passed on correctly to the secondary or service container, contexts or project specific environment variables won't be interpolated for non-primary containers.
```

Comment:
```suggestion
**Note**: While hard-coded environment variable values will be passed on correctly to the secondary or service container, contexts or project specific environment variables will not be interpolated for non-primary containers.
```
We don't currently have this in our own style guide, but I'll follow Microsoft's style guide here and use a hyphen.
We do avoid contractions when we can! https://circleci.com/docs/style/style-voice/#word-choices